src/main/java/io/jenkins/plugins/jfr/JfrAction.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.jfr;
+
+import hudson.model.Action;
+import java.util.Collection;
+import java.util.Collections;
+import org.jspecify.annotations.NonNull;
+
+public class JfrAction implements Action {
+
+    @Override
+    @NonNull
+    public String getIconFileName() {
+        return "document.png";
+    }
+
+    @Override
+    @NonNull
+    public String getDisplayName() {
+        return "Java Flight Recorder";
+    }
+
+    @Override
+    @NonNull
+    public String getUrlName() {
+        return "jfr";
+    }
+
+    @NonNull
+    public Collection<JfrConfig> getSessions() {
```

Comment:
Please make this a Stapler WebMethod that RequiresGET. It should serialize json using an injected objectmapper and write that to the Stapler response's writer.